src/client/unified-form/cover-tab/isbn-field.tsx

```diff
@@ -52,6 +53,7 @@ function mapStateToProps(rootState:State):ISBNStateProps {
 }
 
 function mapDispatchToProps(dispatch):ISBNDispatchProps {
+	let gett:dispatchResultProps|null;
```

Comment:
Maybe a better name for this one would be:
```suggestion
	let autoAddedISBN:dispatchResultProps|null;
```